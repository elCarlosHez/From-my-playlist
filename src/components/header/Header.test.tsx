import { create } from "react-test-renderer";
import { Header } from "./";

describe('<Header /> tests', () => {
  it('Render the Header component.', () => {
    const component = create(<Header />);
    const json = component.toJSON();
    expect(json).toBeDefined();
    expect(json).toMatchSnapshot();
  });
});

