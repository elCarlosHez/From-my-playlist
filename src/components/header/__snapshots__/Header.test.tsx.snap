// Vitest Snapshot v1

exports[`<Header /> tests > Render the Header component. 1`] = `
<header
  className="flex justify-center items-center flex-col"
>
  <h1>
    <img
      alt="From my music"
      className="max-w-full"
      src="/src/assets/logo.png"
    />
  </h1>
</header>
`;
